atlascope/core/models/tour.py

```diff
@@ -0,0 +1,16 @@
+# from typing import List
+from uuid import uuid4
+
+from django.db import models
+from rest_framework import serializers
+
+
+class Tour(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
```

Comment:
If #120 is approved, you'll want to strike this line.

Reply:
resolved with a29845873608f22d5ba2d213129b54560c2522ae